cli/testdata/coder_stat_cpu_--help.golden

```diff
@@ -0,0 +1,16 @@
+Usage: coder stat cpu [flags]
+
+Show CPU usage, in cores.
+
+[1mOptions[0m
+      --host bool
+          Force host CPU measurement.
+
+  -o, --output string (default: text)
+          Output format. Available formats: text, json.
+
+      --prefix string
+          Unit prefix.
```

Comment:
No default here? How will the user know what options are available?

PS. Intuitively I would find `--unit` more declarative when reading the flag. As a flag on the CLI, reading `--prefix` kind of suggests the output will be prefixed with something which isn't the case.

Reply:
Might be time to just use `clibase.EnumOf`

Reply:
Argument is gone, so no need.